src/js/angular/export/controllers.js

```diff
@@ -112,6 +131,10 @@ exportCtrl.controller('ExportCtrl',
                 }
             };
 
+            $scope.formatNumberForLanguage = (number) => {
```

Comment:
I don't like this. Better implement a filter (pipe) and use it in templates instead of calling this function